src/Entity/CategoryManager.php

```diff
@@ -155,13 +120,6 @@ public function getRootCategoriesForContext(ContextInterface $context = null)
         return $this->categories[$context->getId()];
     }
 
-    /**
-     * @todo NEXT_MAJOR: remove this method.
```

Comment:
Shouldn’t we keep this?

Reply:
IMHO we don't need this. We just need to port all `@method` methods

Reply:
For adding yes, but this is about removing a method

Reply:
Oh, you're right

Reply:
There are others too IIRC

Reply:
Fixed